api.json

```diff
@@ -1171,14 +1171,11 @@
         }
       },
      "SyncStatus": {
-       "description":"SyncStatus is used to provide additional context about an implementation's sync status. It is often used to indicate that an implementation is healthy when it cannot be queried  until some sync phase occurs. If an implementation is immediately queryable, this model is often not populated.",
+       "description":"SyncStatus is used to provide additional context about an implementation's sync status. This object is often used by implementations to indicate healthiness when block data cannot be queried until some sync phase completes or cannot be determined by comparing the timestamp of the most recent block with the current time.",
        "type":"object",
-       "required": [
-         "current_index"
-        ],
        "properties": {
          "current_index": {
-           "description":"CurrentIndex is the index of the last synced block in the current stage.",
+           "description":"CurrentIndex is the index of the last synced block in the current stage. This is a separate field from current_block_identifier in NetworkStatusResponse because blocks with indexes up to and including the current_index may not yet be queryable by the caller. To reiterate, all indexes up to and including current_block_identifier in NetworkStatusResponse must be queryable via the /block endpoint (excluding indexes less than oldest_block_identifier).",
```

Comment:
small nit: `indexes` -> `indices`, but not a big deal

Reply:
🤦 I always did better with numbers